release-1.8/release-1.8.md

```diff
@@ -0,0 +1,81 @@
+# Proposed timeline for 1.8
+We will begin the 1.8 release the first week of July 2017.
+
+The proposal below is similar in layout to the 1.7 plans with some
+modifications:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+
+## 1.8 Release Schedule
+- July 5 (Weds) coding start (7w)
+- July 12th (Weds): v1.8.0-alpha.1
+- July 26th (Weds): v1.8.0-alpha.2
+- Aug 1 (Tues): features repo freeze
```

Comment:
suggest, just say "Feature Freeze" here and define later down in the doc.
The term has been used for the past 5-ish releases so it's not new anyway